### PR TITLE
Fix dead links in doc for Mastodon API

### PR DESCRIPTION
**Pull request by stom79**
Recently, Mastodon changed its API doc. This PR is for updating links

### CODE REVIEW

File: doc/API-Mastodon.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Mastodon API
 
 * [Home](help)
-  * [Using the APIs](help/api)
+  * [Using the APIs](https://docs.joinmastodon.org/api/)
```
Comment (tobiasd) [inline doc/API-Mastodon.md]: This link is meant to be the link to the Friendica API documentation, not the Mastodon API.

Comment (stom79) [reply doc/API-Mastodon.md]: Yes, sorry.

**Comment by tobiasd** [comment]
The information of the new location is already valid right? So this should be in the 2019.12 release and does not have to wait for March.

**Comment by tobiasd** [comment]
Ok, simply changing the base pulls in too many changes from the develop branch :-/ So either you need to create a new pull request against the RC branch, with changes based on the RC branch, or the documentation will only be updated in the March release of Friendica.

**Comment by stom79** [comment]
I will redo the PR from the correct branch.

**Comment by stom79** [comment]
New PR: https://github.com/friendica/friendica/pull/7982

**Pull request closed without merging**